e2e/setup/auth.setup.ts

```diff
@@ -1,17 +1,21 @@
 import { test as setup } from "@playwright/test";
 
+const loginDomain = process.env.KEYCLOAK_ENABLED
```

Comment:
```suggestion
const loginDomain = process.env.KEYCLOAK_ENABLED === 'true'
```

env variables are strings